package.json

```diff
@@ -22,9 +22,15 @@
     },
     "homepage": "https://github.com/FACxBeamery/grads-final-project#readme",
     "dependencies": {
+        "@hapi/joi": "^16.1.8",
         "airbnb": "0.0.2",
         "eslint-config-airbnb": "^18.0.1",
         "eslint-config-prettier": "^6.7.0",
+        "joi": "^14.3.1",
```

Comment:
why is there 2 types of joi. Are both necessary?

Reply:
in fact neither of these are needed in the root package.json - I've removed them in the next commit